src/Configuration.php

```diff
@@ -36,6 +39,12 @@ public function __construct(
          * Does the mapper should throw an exception if the target is read-only.
          */
         public bool $allowReadOnlyTargetToPopulate = false,
+        /**
+         * Classes with unknown field names, implemeting ArrayAccess
+         *
+         * @var list<class-string<\ArrayAccess<string, mixed>>> $arrayAccessClasses
+         */
+        public array $arrayAccessClasses = [\ArrayObject::class, Document::class, BSONDocument::class],
```

Comment:
Could we inject theses classes only if they exists ?
Like adding a `if (class_exists(...))` in the construct instead of always having them ?
(MongoDB ones)